admin/tabs/general/alertBanner/AlertBannerPanel.ts

```diff
@@ -188,3 +201,67 @@ const previewPanel = hoistCmp.factory<AlertBannerModel>(({model}) => {
         ]
     });
 });
+
+//------------------
+// Preset Menu
+//------------------
+const presetMenu = hoistCmp.factory<AlertBannerModel>({
+    render({model}) {
+        const savedPresets = model.savedPresets,
+            {message, intent, iconName, enableClose} = model.formModel.values,
+            items = [];
+
+        if (isEmpty(savedPresets)) {
+            items.push(menuItem({text: 'No presets saved...', disabled: true}));
+        } else {
+            items.push(...savedPresets.map(preset => presetMenuItem({preset})));
+        }
+
+        items.push(
+            menuDivider(),
+            menuItem({
+                icon: Icon.add({intent: 'success'}),
+                disabled:
+                    !model.formModel.fields.message.value ||
+                    model.isPreset({message, intent, iconName, enableClose}),
```

Comment:
We shouldn't need to pass this method the current values - it can be a getter on the model - something like `get currentValuesSavedAsPreset() {}` - it would a good use of the `@computed` decorator - we can review what that is and why.